pyaedt/application/Variables.py

```diff
@@ -431,13 +435,13 @@ def decompose_variable_value(variable_value):
 class VariableManager(object):
     """VariableManager class.
 
-    This class manages the design properties and projecct variables. Design properties
+    This class provides for managing design properties and project variables. Design properties
     are the local variables in a design. Project variables are defined at the project 
     level and start with ``$``. 
     
-    This class provides access to all or a subset of the variables. Manipulation
+    This class provides access to all variables or a subset of the variables. Manipulation
     of the numerical or string definitions of variable values is provided in the
-    'Variable' class.
+    `Variable` class.
```

Comment:
```suggestion
    :class:`Variable` class.
```
A bare class reference works here since the class is in the same module as the method referring to it.